tests/testing_support/fixtures.py

```diff
@@ -2055,6 +2055,22 @@ def _validate_transaction_error_trace_count(wrapped, instance, args,
     return _validate_transaction_error_trace_count
 
 
+def validate_transaction_slow_sql_count(num_slow_sql):
```

Comment:
Is this validator one we should also put in its own file to clear up fixtures.py?

Reply:
Yep, good call.